cluster_orchestrator/cluster-manager/mongodb_client.py

```diff
@@ -110,6 +110,8 @@ def mongo_aggregate_node_information(TIME_INTERVAL):
     cumulative_memory_in_mb = 0
     number_of_active_nodes = 0
     technology = []
+    architecture = []
+    aggregation_per_architecture = {}
```

Comment:
What is the difference between architecture and aggregation_per_architecture?

Reply:
The architecture contains only the architecture names and the aggregation_per_architecture contains
the actual hardware information. Should they be combined?

Reply:
Can you give me an example? I did not get it sorry 😂

Reply:
In the end the fields would look something like this:
```json
{
   "architecture":[
      "amd64",
      "aarch64"
   ],
   "aggregation_per_architecture":{
      "amd64":{
         "cpu_percent":"0.1",
         "cpu_cores":"10",
         "..."
      },
      "aarch64":{
         "cpu_percent":"0.8",
         "cpu_cores":"20",
         "..."
      }
   }
}
```

Reply:
@giobart Since the filed is actually never used and also redundant I removed it with commit [d0aa5d1](https://github.com/oakestra/oakestra/pull/163/commits/d0aa5d1c2d0082c693e1c24c44b387500fb90834)